doc/source/api/executors/awslambda.rst

```diff
@@ -9,11 +9,15 @@ With this executor, users can execute tasks (electrons) or entire lattices using
 to use this plugin for electrons that are expected to be short lived and low in compute intensity. This plugin can also be used
 for workflows with several short lived embarassingly parallel tasks aka. horizontal workflows.
 
+.. note::
+    Due to the isolated nature of AWS Lambda, the packages available on that environment are limited. This means that only the modules that
```

Comment:
Can you add in this as well @kessler-frost ? https://aws.amazon.com/premiumsupport/knowledge-center/lambda-python-package-compatible/

Reply:
Updated 👍 .